x-pack/filebeat/module/misp/threat/config/pipeline.js

```diff
@@ -48,14 +49,17 @@ var threat = (function () {
                 } else {
                     asn = v;
                 }
-                attackPattern = '[' + 'as.number = ' + '\'' + asn + '\'' + ']';
+                attackPattern = '[' + 'as:number = ' + '\'' + asn + '\'' + ']';
+                attackPatternKQL = 'as.number: ' + asn;
```

Comment:
In ECS we have a `source.as.number` and a `destination.as.number`, but not an `as.number`. So I think it should query both fields.